torch/_dynamo/output_graph.py

```diff
@@ -1345,203 +1346,6 @@ def remove_unused(node):
                     # Make sure we delete later occurrences of the same symbol
                     used_symbols.remove(symbol)
 
-    # TODO: this is a generic pass that should live outside of Dynamo
-    def insert_deferred_runtime_asserts(self, root, name) -> None:
-        """
```

Comment:
I think this pass should live in AOTAutograd, not just in export. With this change, torch.compile wouldn't be running the asserts anymore. (cc: @ezyang )

Reply:
Oh ok makes sense.